android/app/src/main/java/com/nt4f04und/sweyer/MainActivity.java

```diff
@@ -1,50 +1,17 @@
 package com.nt4f04und.sweyer;
 
 import android.content.Context;
-import android.content.Intent;
-import android.os.Bundle;
 
-import androidx.annotation.Nullable;
+import androidx.annotation.NonNull;
 
-import com.nt4f04und.sweyer.channels.ContentChannel;
-import com.nt4f04und.sweyer.handlers.GeneralHandler;
 import com.ryanheise.audioservice.AudioServicePlugin;
 
 import io.flutter.embedding.android.FlutterActivity;
 import io.flutter.embedding.engine.FlutterEngine;
-import io.flutter.plugin.common.BinaryMessenger;
 
 public class MainActivity extends FlutterActivity {
```

Comment:
At this point we could just extend from [`AudioServiceActivity`](https://github.com/ryanheise/audio_service/blob/minor/audio_service/android/src/main/java/com/ryanheise/audioservice/AudioServiceActivity.java).